pkgs/top-level/all-packages.nix

```diff
@@ -9144,7 +9144,9 @@ in
   foreman = callPackage ../tools/system/foreman { };
   goreman = callPackage ../tools/system/goreman { };
 
-  framac = callPackage ../development/tools/analysis/frama-c { };
+  framac = callPackage ../development/tools/analysis/frama-c { 
+    ocamlPackages = ocaml-ng.ocamlPackages_4_05;
```

Comment:
Is something wrong with 4.06?

Reply:
I had some reason but I can't remember it, and it works now with 4.06.